src/main/java/core/basesyntax/serviceimpl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    @Override
+    public void write(String finalReport, File file) {
```

Comment:
The method parameter 'File file' could lead to hardcoding paths when using this service. It's better to accept a String filePath parameter and create the File object inside the method using the provided path.